cellpack/autopack/Environment.py

```diff
@@ -151,9 +143,7 @@ def __init__(self, config=None, recipe=None):
 
         # saving/pickle option
         self.saveResult = "out" in config
-        self.out_folder = RecipeLoader.create_output_dir(
-            config["out"], name, config["place_method"]
-        )
+        self.out_folder = create_output_dir(config["out"], name, config["place_method"])
```

Comment:
I just moved this to utils because it was a static method anyway, and now we're not importing `RecipeLoader`, which seemed like a bad dependency for `Environment` to have